### ISSUE TITLE
Should interpylate be exported?

### ISSUE THREAD

**Issue by andrie** [RevolutionAnalytics/webapi#2]
It is exported in the NAMESPACE, but has no documentation.

**Comment by piccolbo** [comment]
Yes, it allows free form url design when query or path (django-style) are not sufficient. So users need to specify it as params.encoding arg, unless they have their own function, which they can also supply. What is does is a limited python style  interpolation, only by named arguments.
